index.ts

```diff
@@ -36,6 +59,76 @@ app.post('/wh/updateDocsJson', async (req, res) => {
 	})
 })
 
+app.post('/tutorial/feedback', feedbackRateLimit, async (req, res) => {
+	const body = JSON.parse(JSON.stringify(req.body)) as FeedbackRequestBody
+	// validation of request body
+	if (
+		typeof body.turnstileToken !== "string" ||
+		// type of string should be "up" or "down" but it's checked later
+		typeof body.feedback !== "string" ||
+		typeof body.route !== "string"
+  	)
+    return res
```

Comment:
formatted a little weird here

Reply:
yeah, big L for prettier

Reply:
it looks good on my part: ![](https://img.srizan.dev/Code_9a25ZHmqNG.png)